code/index.html

```diff
@@ -2,21 +2,130 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=7">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
       <!-- Some kind of logo and navbar -->
-    </header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy"> News</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">Contact</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">About</a></li>
+      </ul>
+    </nav>
```

Comment:
It seems like you have different indentation and spacing for the same elements here. It's easier to read if you use the same indentation and spacing everywhere: 2 spaces for indentation is technigos "rule". And then make sure that the closing tag is on the same level as the opening tag.